### DISCUSSION TITLE
How to entirely remove default mappings?

### DISCUSSION THREAD

**Discussion by alankopetman** [NvChad/NvChad#908] [Q&A] [answered]
Is it possible to entirely remove default mappings found in chadrc.lua? Specifically the terminal related mappings. I use LazyGit and use `jk` to navigate around and it's causing me to exit terminal mode.

**Comment by siduck** [comment]
https://nvchad.github.io/config/Custom%20config#use-default-mapping-for-another-command

**Reply by Ahmad-Alam** [reply]
Hi, is there a newer link for this? I'm looking for an aswer to the same question.

**Reply by siduck** [reply, marked as answer]
@Ahmad-Alam  just dont include nvchad.mappings module!  its loading in your lua/mappings.lua file, just remove that line
